integrationtests/src/test/resources/kafka_jaas.conf

```diff
@@ -0,0 +1,7 @@
+//
+// Copyright Kroxylicious Authors.
+//
+// Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+//
+
+// Intentionally empty
```

Comment:
Explain why it's empty/what is responsible for adding the extra.

Reply:
The file was required for bitnami. Bitnami has gone, so it is removed too.